typescript/helloworld/src/app/app.ts

```diff
@@ -64,16 +75,19 @@ export class HelloWorldApp<
     return { sent: sent.toNumber(), received: received.toNumber() };
   }
 
-  async stats() {
-    const entries = await Promise.all(
+  async stats(): Promise<Record<string, Record<string, Counts>>> {
```

Comment:
I tried that, but because of reasons it seemed to be an issue. Let me check... I just don't like having to add another cast but it is more correct so done.